tests/executors/test_executor_loader.py

```diff
@@ -301,3 +302,21 @@ def test_validate_database_executor_compatibility_sqlite(self, monkeypatch, exec
         monkeypatch.delenv("_AIRFLOW__SKIP_DATABASE_EXECUTOR_COMPATIBILITY_CHECK")
         with expectation:
             ExecutorLoader.validate_database_executor_compatibility(executor)
+
+    def test_load_executor(self):
```

Comment:
I touched this method above and noticed it had no tests (at least I can't find them) so I added a couple new ones here.

Reply:
Nice addition